src/common/Pagination/styled.js

```diff
@@ -1,5 +1,5 @@
 import styled, { css } from "styled-components";
-import { ReactComponent as Vector } from "./vector.svg";
+import { ReactComponent as Vector } from "./Vector.svg";
```

Comment:
Sorry, but this change causing error, because imports now don't match, Application doesn`t get up.